packages/TelegramClient-Core.package/TCCGroup.class/instance/isGroup.st

```diff
@@ -0,0 +1,4 @@
+accessing
+isGroup
+
+	^ true
```

Comment:
We thought it was weird to have a method isGroup for elements of class Group but I guess it is fine for now.

Reply:
> We thought it was weird to have a method isGroup for elements of class Group but I guess it is fine for now.

It is a subclass of ``TCCChat` which has an `isGroup` method

Reply:
Hm I dont know; We found that `isGroup` is more descriptive and better understandable than `isNotPrivate`.